### PR TITLE
Update metadata.json

### PR DESCRIPTION
**Pull request by astahjmo**
New logo for this site

### CODE REVIEW

File: websites/A/Animes House/dist/metadata.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   },
   "url": "animeshouse.net",
   "version": "1.0.3",
```
Comment (callumok2004) [inline websites/A/Animes House/dist/metadata.json:13]: Please update to 1.0.4 as bas requested, reacting to his post does not do much

Comment (Alanexei) [inline websites/A/Animes House/dist/metadata.json:13]: Increase the version

Comment (astahjmo) [reply websites/A/Animes House/dist/metadata.json]: corrected, I ended up forgetting this detail, sorry

Comment (astahjmo) [reply websites/A/Animes House/dist/metadata.json]: corrected, I ended up forgetting this detail, sorry

**Pull request merged**